src/walletcore/userWallet.ts

```diff
@@ -85,14 +46,23 @@ export class UserWallet {
         return new UserSecretKey(seed);
     }
 
-    /**
-     * Will take about:
-     *  - 80-90 ms in Node.js, on a i3-8100 CPU @ 3.60GHz
-     *  - 350-360 ms in browser (Firefox), on a i3-8100 CPU @ 3.60GHz
-     */
-    private static generateDerivedKey(password: Buffer, salt: Buffer, kdfparams: ScryptKeyDerivationParams): Buffer {
-        const derivedKey = scryptsy(password, salt, kdfparams.n, kdfparams.r, kdfparams.p, kdfparams.dklen);
-        return derivedKey;
+    static ecFromJSON(keyfileObject: any): EncryptedData {
```

Comment:
What does the prefix `ec` mean?

Reply:
It was a typo, should have been `ed` for "Encrypted Data"